examples/fft_vs_geom_movie.py

```diff
@@ -0,0 +1,348 @@
+
+Some suggested command lines, appropriate for LSST:
+
+Optics only simulation:
+  $ python fft_vs_geom_movie.py --diam 8.36 --obscuration 0.61 --sigma 0.05 --nlayers 0
+
+Atmosphere only simulation:
+  $ python fft_vs_geom_movie.py --diam 8.36 --obscuration 0.61 --sigma 0.0 --nlayers 6 --size 3.0
+
+"""
+
+
+import numpy as np
+import galsim
+
+try:
+    import matplotlib
+    import matplotlib.pyplot as plt
+    import matplotlib.animation as anim
+except ImportError:
+    raise ImportError("This demo requires matplotlib!")
+from distutils.version import LooseVersion
+if LooseVersion(matplotlib.__version__) < LooseVersion('1.2'):
+    raise RuntimeError("This demo requires matplotlib version 1.2 or greater!")
+
+try:
+    from astropy.utils.console import ProgressBar
+except ImportError:
+    raise ImportError("This demo requires astropy!")
+
+def simple_moments(img):
+    """Compute unweighted 0th, 1st, and 2nd moments of image.  Return result as a dictionary.
+    """
+    array = img.array
+    scale = img.scale
+    N = array.shape[0]
+    # x = y = np.arange(array.shape[0])*scale
+    x = y = np.arange(array.shape[0])-(N/2)
+    y, x = np.meshgrid(y, x)
+    I0 = np.sum(array)
+    Ix = np.sum(x*array)/I0
+    Iy = np.sum(y*array)/I0
+    Ixx = np.sum((x-Ix)**2*array)/I0
+    Iyy = np.sum((y-Iy)**2*array)/I0
+    Ixy = np.sum((x-Ix)*(y-Iy)*array)/I0
+    return dict(I0=I0, Ix=Ix, Iy=Iy, Ixx=Ixx, Iyy=Iyy, Ixy=Ixy)
+
+def ellip(mom):
+    """Convert moments dictionary into dictionary with ellipticity (e1, e2) and size (rsqr).
+    """
+    rsqr = mom['Ixx'] + mom['Iyy']
+    return dict(rsqr=rsqr, e1=(mom['Ixx']-mom['Iyy'])/rsqr, e2=2*mom['Ixy']/rsqr)
+
+def make_movie(args):
+    rng = galsim.BaseDeviate(args.seed)
+    u = galsim.UniformDeviate(rng)
+    # Generate 1D Gaussian random fields for each aberration.
+    t = np.arange(-args.n/2, args.n/2)
+    corr = np.exp(-0.5*t**2/args.ell**2)
+    pk = np.fft.fft(np.fft.fftshift(corr))
+    ak = np.sqrt(2*pk)
+    phi = np.random.uniform(size=(args.n, args.jmax))
+    zk = ak[:, None]*np.exp(2j*np.pi*phi)
+    aberrations = args.n/2*np.fft.ifft(zk, axis=0).real
+    measured_std = np.mean(np.std(aberrations, axis=0))
+    aberrations *= args.sigma/measured_std
+    aberrations -= np.mean(aberrations, axis=0)
+
+    # Generate an atmosphere.  Same procedure as in psf_wf_movie.py
+    Ellerbroek_alts = [0.0, 2.58, 5.16, 7.73, 12.89, 15.46]  # km
+    Ellerbroek_weights = [0.652, 0.172, 0.055, 0.025, 0.074, 0.022]
+    Ellerbroek_interp = galsim.LookupTable(Ellerbroek_alts, Ellerbroek_weights,
+                                           interpolant='linear')
+    alts = np.max(Ellerbroek_alts)*np.arange(args.nlayers)/(args.nlayers-1)
+    weights = Ellerbroek_interp(alts)  # interpolate the weights
+    weights /= sum(weights)  # and renormalize
+    max_speed = 20  # Pick (an arbitrary) maximum wind speed in m/s.
```

Comment:
I went ahead and made `max_speed` a command line argument, like most of the other keyword arguments.  I also added some text to the file docstring indicating that you can run `python fft_vs_geom_movie.py --help` to see all the options and defaults.